src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidget.java

```diff
@@ -172,21 +194,38 @@ public void configure(EntityBundle bundle, Long versionNumber, boolean canEdit,
 		this.actionMenu = actionMenu;
 		this.entityTypeDisplay = EntityTypeUtils.getFriendlyEntityTypeName(bundle.getEntity());
 		reconfigureState();
+		showEditorIfEditableAndEmpty();
 	}
 
 	private void reconfigureState() {
+		initializeQuery(() -> initSimpleAdvancedQueryState());

```

Comment:
Initialize the query first so we can reconfigure actions based on the query state.